### ISSUE TITLE
Port conflict in client docker setup

### ISSUE THREAD

**Issue by jjtt** [alievk/avatarify#310]
https://github.com/alievk/avatarify/blob/e3302b94059f0dfb6b32b82ff2c1e3ca3834cbb3/run.sh#L112

Using 5557 twice

**Comment by mintmaker** [comment]
Do you mean you get a conflict when running client and server on the same machine? In that case you should add the `--is-local-client` flag (as stated in the wiki).

**Comment by jjtt** [comment]
No, I mean that the script tries to use the same port 5557 twice : `-p 5557:5554 -p 5557:5558`

**Comment by mintmaker** [comment]
The first one can be safely deleted as it is not used anywhere. Not sure if the second one is right though. I will look into this as soon as I find the time.

**Comment by mintmaker** [comment]
Both can be deleted as the ports are already exposed in the Dockerfile. @alievk If you like you can just remove the whole if statement in the run.sh there. Otherwise I can make a pr. Also in #296 a mistake in the readme was reported (you would just need to replace a 'hardly' with 'highly'). Might be easier for you to do both things directly than with a pr.
